biogeochem/FatesSoilBGCFluxMod.F90

```diff
@@ -1182,7 +1182,7 @@ function ECACScalar(ccohort, element_id) result(c_scalar)
     integer, parameter :: downreg_logi   = 2
     integer, parameter :: downreg_CN_logi = 3
     
-    integer, parameter :: downreg_type = downreg_linear
+    integer, parameter :: downreg_type = downreg_CN_logi
```

Comment:
@jenniferholm have you been getting better results with the logistic feedback mechanism? 
If so, lets submit this change in an other PR to maintain bit-for-bit answers with baseline tests

Reply:
@rgknox - I didn't find the differences between linear and CN_logistic to be that large, or at least not make the results any "better".  I think this was left here from my last test, and I can revert it back to linear to maintain bit-for-bit.  

The same goes for the N and P efflux code changes that I've been testing with. Since we've been supplying such a large amount of nutrients during testing, I turned on the n_efflux and p_efflux, so that it does not go back into storage.  I'm not sure what the default should be for this, and I can defer to you (or Charlie or others).  My only two-cents is that maybe it does seem like a good idea to have defaulted "turned on" so that we won't get this really high accumulation of nutrients into storage by accident, when testing.....

Reply:
Regarding your second point.  If the stores of N and P are at or above the maximum holding capacity, the need function that drives uptake on the following model steps should drop to 0 in all cases.  This includes RD, ECA and prescribed.  So stores should not start growing without control (should being the key word, hehe).  I think the only time I've seen stores get way above the target maximum is when I've been using poorly calibrated ECA parameters, which will overshoot uptake from soils with high availability by orders of magnitude. But in that situation, we are so off the mark in terms of what we would expect from nature, I don't think it matters if we hold the excess in storage or dump it to a labile pool.

Reply:
@rgknox - I went ahead and reverted these back to the default values and settings to be easier, and b4b.  So back to "downreg_linear" and also the n_efflux and p_efflux back to storage.  That totally makes sense that the plant need function will drive the uptake, and thus stop any excess uptake of the stores of N and P are at it's holding capacity, so limit any really high uptake and excess.